src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -211,7 +211,11 @@ class AddressValidationModal extends React.Component {
           status="warning"
           headline={addressValidationMessage.headline}
         >
-          <p>{addressValidationMessage.modalText}</p>
+          <p>
+            {addressValidationMessage.modalText(() => {
+              this.props.openModal(addressValidationType, addressFromUser);
+            })}
+          </p>
```

Comment:
Not sure, but I think we could use the JSX syntax here? `<addressValidationMessage.modalText editFunction={ this.props.openModal}/>`. I don't know, could make it feel more React-y.

Reply:
I'll explore that.